test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
♻️ Adding files to `.gitignore` file is a good practice and the standard way to avoid shipping test/private/development-only files that you don't want to push to your repo. Even though you already set this file inside the `.gitignore` it is still present in your repo.

- [ ] Please make sure that once you add this file to the `.gitignore` it is not tracked anymore by git. Kindly take a look at [this post](https://timmousk.com/blog/git-rm-cached/). It might help you. Also do not hesitate to search the [git documentation](https://git-scm.com/docs/git-rm) 📖📖

Reply:
Hi, @rskinnerc, I have removed the test.md file from the repository using the git rm --cached test.md command. Please check if I did it correctly.